WalletWasabi.Gui/Controls/WalletExplorer/ClosedWalletViewModel.cs

```diff
@@ -0,0 +1,73 @@
+using AvalonStudio.Extensibility;
+using ReactiveUI;
+using Splat;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Threading;
+using WalletWasabi.Gui.Helpers;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class ClosedWalletViewModel : WalletViewModelBase
```

Comment:
Why is it better to have two viewmodels for one wallet? How about having one and use the state of the wallet if it is opened or not?

Reply:
Because we want to use different templates, so you can have different context menus, otherwise you need a whole load of logic, converters and it gets un-maintainable.